### PR TITLE
*: add some logs to help debug error.

### PR DESCRIPTION
**Pull request by asddongmen**
### What problem does this PR solve? 
Due to the lack of logs, this issue #2116  are difficult to locate the root cause.
The purpose of the PR is to better locate such problems.

### What is changed and how it works?
1.add some log
2.correct some typos

### Check List 

Tests 

 - No code

Code changes

 - None

Side effects

 - None

Related changes

 - Need to cherry-pick to the release branch

### Release note

- No release note

### CODE REVIEW

File: cdc/model/owner.go [hunk 1]
```diff
@@ -238,6 +241,7 @@ func (ts *TaskStatus) RemoveTable(id TableID, boundaryTs Ts, isMoveTable bool) (
 		return nil, false
 	}
 	delete(ts.Tables, id)
+	log.Info("remove a table", zap.Int64("table-id", id))
```
Comment (amyangfei) [inline cdc/model/owner.go:244]: can also log `boundaryTs` and `isMoveTable`

File: cdc/model/owner.go [hunk 2]
```diff
@@ -262,6 +266,7 @@ func (ts *TaskStatus) AddTable(id TableID, table *TableReplicaInfo, boundaryTs T
 		return
 	}
 	ts.Tables[id] = table
+	log.Info("add a table", zap.Int64("table-id", id))
```
Comment (amyangfei) [inline cdc/model/owner.go:269]: ditto

File: cdc/owner/feed_state_manager.go [hunk 3]
```diff
@@ -111,7 +111,6 @@ func (m *feedStateManager) handleAdminJob() (jobsPending bool) {
 		}
 		m.shouldBeRunning = false
 		jobsPending = true
-		m.patchState(model.StateRemoved)
```
Comment (amyangfei) [inline cdc/owner/feed_state_manager.go:114]: Why is this line removed

Comment (asddongmen) [reply cdc/owner/feed_state_manager.go:114]: I confirmed with @leoppro , this line is redundant.
```golang
		// remove changefeed info and state
		m.state.PatchInfo(func(info *model.ChangeFeedInfo) (*model.ChangeFeedInfo, bool, error) {
			return nil, true, nil
		})
		m.state.PatchStatus(func(status *model.ChangeFeedStatus) (*model.ChangeFeedStatus, bool, error) {
			return nil, true, nil
		})
```
The above two lines of code will directly delete changefeed.

File: cdc/model/owner.go (hunk 1 above)
Comment (asddongmen) [reply cdc/model/owner.go]: done

File: cdc/model/owner.go [hunk 4]
```diff
@@ -18,7 +18,10 @@ import (
 	"fmt"
 	"math"
 
+	"go.uber.org/zap"
+
 	"github.com/pingcap/errors"
```
Comment (overvenus) [inline cdc/model/owner.go:23]:
```suggestion
	"go.uber.org/zap"
	"github.com/pingcap/errors"
```

**Review by overvenus — COMMENTED** [review]
rest LGTM

**Comment by ben1009** [comment]
make: *** [fmt] Error 1
./cdc/model/owner.go

**Review by overvenus — APPROVED** [review]

File: cdc/model/owner.go [hunk 5]
```diff
@@ -238,6 +240,7 @@ func (ts *TaskStatus) RemoveTable(id TableID, boundaryTs Ts, isMoveTable bool) (
 		return nil, false
 	}
 	delete(ts.Tables, id)
+	log.Info("remove a table", zap.Int64("table-id", id), zap.Uint64("boundaryTs", boundaryTs), zap.Bool("isMoveTable", isMoveTable))
```
Comment (3AceShowHand) [inline cdc/model/owner.go:243]: how about rename `table-id` to `tableID`, to keep log format pattern the same, just like `isMoveTable` or `ResolvedTs`

Comment (asddongmen) [reply cdc/model/owner.go]: done

**Review by amyangfei — APPROVED** [review]
